packages/core/src/components/DatePicker/DatePicker.tsx

```diff
@@ -57,6 +62,10 @@ export interface DatePickerProps extends VibeComponentProps {
    * If true, enables date range selection mode.
    */
   range?: boolean;
+  /**
+   * Custom phrases for accessibility and aria-labels.
+   */
+  phrases?: DayPickerPhrases;
```

Comment:
Are there any properties that we'd want to expose? I'm wondering if it's better to just expose `aria-label` and keep the rest the same. And if so, maybe spread them to more descriptive props. WDYT?

Reply:
we could definitely expose them in a different way – I opted to expose exactly the same thing as react-dates, because we already use the defaultPhrases, and this way the two types coincide exactly, but I'm open to suggestions here, whatever you feel is the right API for this.

Reply:
Cool let's go with full flexibility